app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -332,6 +340,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
   $scope.providerTypeChanged = function() {
     if ($scope.emsCommonModel.ems_controller === 'ems_container') {
       $scope.emsCommonModel.default_api_port = "8443"; // TODO: correct per-type port
+      $scope.emsCommonModel.metrics_selection = "hawkular_".concat($scope.emsCommonModel.default_metrics_selection);
```

Comment:
most `default_foo` vars here refer to Default endpoint's foo.
to express "default value of metrics_selection" consider `metrics_selection_default` name.
oh, you already called it that in backend.